ui/shared/components/panel/search/GeneBreakdown.jsx

```diff
@@ -39,13 +39,6 @@ const COLUMNS = [
     format: row => (
       <SearchResultsLink
         location={row.geneId}
-        projectFamilies={Object.entries(row.families.reduce((acc, { family }) => {
-          if (!acc[family.projectGuid]) {
-            acc[family.projectGuid] = []
-          }
-          acc[family.projectGuid].push(family.familyGuid)
-          return acc
-        }, {})).map(([projectGuid, familyGuids]) => ({ projectGuid, familyGuids }))}
```

Comment:
Is there any reason that this parameter can be eliminated?

Reply:
yes it was not being used at all, and when I was trying to figure out how the component was working its presence was super confusing